community/CloudflareR2.md

```diff
@@ -0,0 +1,144 @@
+nav_order: 15
+---
+
+# How to host BlueMap on Cloudflare R2
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+## What is Cloudflare R2?
+
+Cloudflare R2 is a service that allows you to host files on Cloudflare's network. With a couple of tweaks, you can host static websites such as BlueMap on Cloudflare R2. Please bear in mind that Cloudflare R2 costs money.
+
+## What do you need for this?
+
+- Cloudflare account
+- Domain you own & is connected with Cloudflare
+- Credit card/prepaid card you have
+- a completely rendered BlueMap map (you can use the CLI or the docker container for that) — this is intended as a one-time upload for a map where further adjustments are unnecessary
+- rclone on the system you've rendered the map on
+
+
+## Adding a domain to Cloudflare
+
+(You can skip this part and go to Setting Zero Trust Dashboard part)
+
+Go to Websites, click on “Add a site” then write your domain name.
+
+1. “Select your plan”
+    - You can choose Free plan at the bottom.
+
+2. “Review DNS records”
+    - You can skip it because we don't need to set a DNS for this method.
+
+3. “Change your nameservers”
+    - Go to the site you bought your domain and change nameservers to Cloudflare's nameservers.
+
+
+## Creating a bucket & uploading files with rclone
+
+1. Go to R2, click on “Create bucket” then write your bucket name.
+
+2. In your new bucket, go to settings.
+
+3. Scroll to Public Access / Custom Domains and click on “Connect Domain”. Enter the domain you want to use for BlueMap and confirm the changes.
+
+4. (optional) Scroll down and edit the “Default Multipart Abort Rule” by clicking on the three dots and then “Edit”. Set the “Multipart Abort Rule” to 1 day.
+
+5. Go back to the R2 overview page and click on “Manage R2 API Tokens”. Click on “Create API Token” and give it a name. Give it “Admin Read & Write” permissions & create the token.  Keep this page open for the rclone configuration.
+
+## Uploading files with rclone
+
+1. Open a terminal and run `rclone config` to configure rclone.
+   1. Choose “n” for new remote.
+   2. Choose a name for the remote, e.g. `R2`.
+   3. Choose `Amazon S3 Compliant Storage Providers` as the storage provider. (As of the time of writing, option `4`.)
+   4. Choose `Cloudflare R2 Storage` as the provider. (As of the time of writing, option `6`.)
+   5. Choose `Enter AWS credentials in the next step.`
+   6. Copy the `Access Key ID` and `Secret Access Key` from the R2 API Token page to the terminal.
+   7. Choose the only possible region.
+   8. Copy the endpoint from the R2 API token page to the terminal.
+   9. Do not edit the advanced config.
+   10. Check if the configuration is correct & confirm.
+
+2. Navigate in the terminal to the directory where your BlueMap *web* files are located. You should be in the directory that contains the file `index.html`.
+
+3. Run the following command to upload the files to your R2 bucket:
+   ```bash
+   rclone sync . R2:drehmal-bluemap/ --transfers=60 --checkers=120 -P
```

Comment:
You've still got your own remote name in here, instead of the one you describe in the explanation below

Reply:
Changed✔ Removed my bucket name (and also mentioned the bucket name `bluemap` as an example further up top.